### DISCUSSION TITLE
I want to change a Help text color with Javascript

### DISCUSSION THREAD

**Discussion by I-like-Mairuuu** [Budibase/budibase#13524] [Help] [answered]
I want to change a Help text color with Javascript 
![Screenshot 2024-04-19 090338](https://github.com/Budibase/budibase/assets/167492552/f2227f71-d74c-4189-9c3c-196fcdd29e45)
I want to change The letter that the arrow points to  red. 😓😓
![Screenshot 2024-04-19 090400](https://github.com/Budibase/budibase/assets/167492552/101fce7b-e6a0-4d00-9949-bd6a1b99185c)

**Comment by mikesealey** [comment, marked as answer]
Hi @I-like-Mairuuu I've been looking into this for you.

I think using Custom Validation might be a better option for what you're trying to achieve. You can find the documentation [here](https://docs.budibase.com/docs/form-validation#custom-validation-rules).

It seems you're using a Data Repeater, which will mean your custom validation rules will have access to the variables you need for comparisons, and the error-message is red. You might want to run this in combination with the helper text.

I hope this helps.
